packages/utilities/src/FabricPerformance.ts

```diff
@@ -0,0 +1,49 @@
+export interface IPerfData {
+  duration: number;
+  timeStamp: number;
+}
+
+export interface IPerfMeasurement {
+  totalDuration: number;
+  count: number;
+  all: IPerfData[];
+}
+
+export interface IPerfSummary {
+  [key: string]: IPerfMeasurement;
+}
+
+const now: () => number = () => (typeof performance !== 'undefined' && !!performance.now) ? performance.now() : Date.now();
+
+export class FabricPerformance {
```

Comment:
is there a particular reason fabric prefers classes with static methods rather than modules?